src/model/client.ts

```diff
@@ -56,15 +70,15 @@ export class CommandoClient extends Client {
     if (typeof filePath === 'string') {
       return walk(filePath).then(files => {
         files.forEach(async (file: string) => {
-          await this.resolveCommand(file)
+          await this.resolveCommand(`${filePath}${filePath.endsWith('\\') ? '' : '\\'}${file}`)
```

Comment:
I dont know if the `\` will work in Linux, so that will need to be tested. Its good enough for now though

Reply:
#2